src/js/hooks/cycleCount/useResolveStep.js

```diff
@@ -161,7 +161,8 @@ const useResolveStep = () => {
 
     const duplicatedItemsValues = Object.values(duplicatedItems);
 
-    return duplicatedItemsValues.filter((group) => group.length < 2).flat();
+    return duplicatedItemsValues
+      .filter((group) => group.length < 2 && group.every((item) => item.countIndex === 0)).flat();
```

Comment:
there was a bug that a custom item added during a recount was visible in the indicator, so unfortunately we have to look for countIndex === 0 here, even though we wanted to avoid relying on them. cc @alannadolny

Reply:
It can be simplified to the code that is included in my recently merged pr